blazor/heatmap-chart/axis.md

```diff
@@ -231,7 +231,238 @@ In heat map, you can place the axis label in an opposite position of its default
 
 ![Blazor HeatMap Chart with Opposed Axis](images/axis/blazor-heatmap-chart-opposed-axis.png)
 
-## Label formatting
+## Axis labels customization
+
+### Customizing the text style
+
+The text style of the axis labels can be customized using the following options available in the `HeatMapXAxisTextStyle`(https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.HeatMap.HeatMapXAxisTextStyle.html) tag for x-axis and `HeatMapYAxisTextStyle`(https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.HeatMap.HeatMapYAxisTextStyle.html) for y-axis.
```

Comment:
Compare the existing content with TS platform, some API links are missing. Add them properly.

Reply:
In EJ2 there is only one property textStyle is common for both x and y-axis. In Blazor for x-axis we are using HeatMapXAxisTextStyle and for y-axis we are using HeatMapYAxisextStyle.

The properties inside the HeatMapXAxisTextStyle and HeatMapYAxisTextStyle are same which link need to add x-axis or y-axis. So, I didn't add API link.